Datadog/Datadog.xcodeproj/project.pbxproj

```diff
@@ -2359,27 +2371,28 @@
 		61133C182423990D00786299 /* Datadog */ = {
 			isa = PBXGroup;
 			children = (
-				61133C192423990D00786299 /* Mocks */,
-				D2956CAE2869D516007D5462 /* DatadogInternal */,
-				61133C412423990D00786299 /* DatadogTests.swift */,
-				61F8CC082469295500FE2908 /* DatadogConfigurationBuilderTests.swift */,
-				61133C382423990D00786299 /* LoggerTests.swift */,
-				61E909F524A32D1C005EA2DE /* GlobalTests.swift */,
-				61C5A89524509BF600DA608C /* TracerTests.swift */,
-				61E917D2246546BF00E6C631 /* TracerConfigurationTests.swift */,
-				61B558CE2469561C001460D3 /* LoggerBuilderTests.swift */,
-				617B953C24BF4D8F00E6F443 /* RUMMonitorTests.swift */,
-				617B954124BF4E7600E6F443 /* RUMMonitorConfigurationTests.swift */,
 				61133C212423990D00786299 /* Core */,
+				61F2724725C9437C00D54BF8 /* CrashReporting */,
+				D2956CAE2869D516007D5462 /* DatadogInternal */,
+				61216278247D20D500AC5D67 /* FeaturesIntegration */,
+				61D3E0DD277B3D6E008BE766 /* Kronos */,
```

Comment:
I just noticed that the `pbxproj` diff looks weird and seems it can bring serious git conflicts with `feature/v2`. Could we limit it (manually) to only include added files?